apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go

```diff
@@ -72,6 +72,27 @@ type CleanupCronJobConfig struct {
 	Schedule string `json:"schedule,omitempty"`
 }
 
+type BackupCronJobConfig struct {
```

Comment:
@dkwon17 
Does it make sense to create a completely new API for backup and don't use devworkspaceoperatorconfig?

Reply:
I am ok to keep it in `DevWorkspaceOperatorConfig`, since we also define a [`CleanupCronJobConfig`](https://github.com/allda/devworkspace-operator/blob/f9297b60437b5fa4eec6d0241ec69e04c1453299/apis/controller/v1alpha1/devworkspaceoperatorconfig_types.go#L52) in the DWOC